andela/healthchecks-prometheus
# Ft configure periodic reports 148476585

#### What does this PR do?
Enable users to choose whether they want to receive email reports and if so be able to select whether to receive the reports daily, weekly or monthly
#### Description of Task to be completed?
- Add radio buttons to report settings form, with options: daily, weekly or monthly(default)
- Add period field to profile model so that user setting can be saved in the database
#### How should this be manually tested?
- Run the command ./manage.py sendreports --loop in the local terminal or heroku terminal
- For testing environment the time in the app can be set to seconds or minutes 
- With app running login to an account and select daily in report settings under account settings (if minute is used, the daily option sets reports to be sent every minute). The user should receive an email every minute
#### What are the relevant pivotal tracker stories?
    #148476585
#### Screenshots
<img width="1289" alt="screen shot 2017-07-28 at 14 27 54" src="https://user-images.githubusercontent.com/13675851/28715772-37e4f8fa-73a2-11e7-8cf9-17722658d3c6.png">
